### PR TITLE
Amplify docs (Getting Started, Storage, Analytics, API)

### PR DESCRIPTION
**Pull request by lawmicha**
https://github.com/undefobj/docs/tree/lawmicha/api-docs/ios
https://github.com/undefobj/docs/tree/lawmicha/api-docs/android

### API (GraphQL/REST)
https://github.com/undefobj/docs/blob/lawmicha/api-docs/ios/api.md
https://github.com/undefobj/docs/blob/lawmicha/api-docs/android/api.md

### Storage
https://github.com/undefobj/docs/blob/lawmicha/api-docs/ios/storage.md
https://github.com/undefobj/docs/blob/lawmicha/api-docs/android/storage.md

### Analytics
https://github.com/undefobj/docs/blob/lawmicha/api-docs/ios/analytics.md
https://github.com/undefobj/docs/blob/lawmicha/api-docs/android/analytics.md

### Getting Started Guide
https://github.com/undefobj/docs/tree/lawmicha/api-docs/ios/start.md
https://github.com/undefobj/docs/tree/lawmicha/api-docs/android/start.md

### CODE REVIEW

File: android/api.md [hunk 1]
```diff
@@ -1,485 +1,231 @@
...
-    }
-    dependencies {
-        // REQUIRED: Typical dependencies
-        implementation 'com.amazonaws:aws-android-sdk-appsync:2.9.+'
-        implementation 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.0'
-        implementation 'org.eclipse.paho:org.eclipse.paho.android.service:1.1.1'
-    }
+```gradle
+{
+    syncEnabled: false
```
Comment (TrekSoft) [inline android/api.md]: Show where in the build gradle this goes

File: android/api.md [hunk 2]
```diff
@@ -1,485 +1,231 @@
...
+You can get the list of items that match a condition that you specify using the `where` parameter in `Amplify.API.query`
 
 ```java
-    private AppSyncSubscriptionCall<OnCreateTodoSubscription.Data> subscriptionWatcher;
-
-    private void subscribe() {
-        OnCreateTodoSubscription subscription = OnCreateTodoSubscription.builder().build();
-        subscriptionWatcher = mAWSAppSyncClient.subscribe(subscription);
-        subscriptionWatcher.execute(subCallback);
+QueryPredicate predicate = Todo.name.eq("MyTodo")
```
Comment (TrekSoft) [inline android/api.md]: Should be Todo.NAME and Todo.DESCRIPTION

File: android/api.md [hunk 3]
```diff
@@ -1,485 +1,231 @@
...
+        .and(Todo.description.eq("description"));
+Amplify.API.query("name", Todo.class, predicate, new ResultListener<GraphQLResponse<Iterable<Todo>>>() {
+    public void onResult(GraphQLResponse<Todo> todoGraphQLResponse) {
+        Log.i("Item", todoGraphQLResponse.getData().getDescription());
```
Comment (TrekSoft) [inline android/api.md]: This onResult is wrong - list returns an Iterable<ToDo> not just a raw <Todo> so you'd have to get the first item from the list.

Comment (lawmicha) [reply android/api.md]: feel free to push directly the changes to this branch

**Pull request merged**